packages/headless/src/controllers/core/quickview/headless-core-quickview.ts

```diff
@@ -31,11 +39,18 @@ export interface QuickviewOptions {
    * The maximum preview size to retrieve, in bytes. By default, the full preview is retrieved.
    */
   maximumPreviewSize?: number;
+
+  /**
+   * Whether to only update the `srcPath` attribute when using `fetchResultContent` rather than updating `content`.
+   * Use this if you are using an iframe with `state.srcPath` as the source url.
+   */
+  onlySrcPath?: boolean;
```

Comment:
Since the purpose of the `srcPath` in the first place is to be used inside of an iframe, I think we should include that in the name. Otherwise, I think we should remove `src` from the name somehow.

For instance:
* `iFrameUrl`/`onlyIFrameUrl`
* `iFramePath`/`onlyIFramePath`
* `iFrameSrc`/`onlyIFrameSrc`
* `downloadUrl`/`onlyDownloadUrl`
* `contentUrl`/`onlyContentUrl`
* etc